src/lib.rs

```diff
@@ -88,9 +62,9 @@ struct CommonOptions {
     #[structopt(long, conflicts_with = "lightbeam")]
     cranelift: bool,
 
-    /// Enable debug output
-    #[structopt(short, long)]
-    debug: bool,
+    /// Enable debug output, currently unused
+    #[structopt(short = "d", long = "debug")]
+    _debug: bool,
```

Comment:
Perhaps we should make the flag now set `RUST_LOG` (if not set)?

Reply:
If `wasmtime=debug` doesn't offer anything useful, I would argue just for `debug` and let users drink from the fire hose; it's always possible for them to tailor it by setting `RUST_LOG` manually.